doc/README.rst

```diff
@@ -0,0 +1,255 @@
+.. _Top:
+
+Generating support keys
+***********************
+
+.. contents:: :local:
+
+Introduction
+============
+
+Each region of FIWARE_ must generate and provided a SSH public key and a GPG
+public key. They are use in the support account of the VMs. As a remainder,
+it is perfectly secure that anybody has access to public keys. Only the private
+keys must remain secret.
+
+Top_
+
+
+Why a SSH key and a GPG key are needed
+======================================
+
+The new base images and all the images based on them, as the migrated GE images,
+have a support account. There are two public keys with an important role in the
+support account:
+
+- a SSH public key: This is the only method to log in the virtual machine with
+  the support account via SSH (the password access is disabled).
+- a GPG public key: This key is used to encrypt the password of the support
+  account. That password is generated inside the VM and after the encryption
+  is printed to the console. The support account password allows to access the
+  VM via console, which is useful if the network is down. Besides, the password
+  is also needed to get full root privileges after connecting to the support
+  account both using console and ssh client due to for security reasons sudo is
+  configured for asking the password.
+
+Another reason to require the support password and encrypt it is to make easy
+enforcing full control and audit of who have admin access to each VM. This is
+because the SSH public is not specific of each VM, but the password yes. There
+are alternatives but they are more complex. For example, do not provide the SSH
+private key to the support staff but use an special designed ssh-agent that audit
+each access. This document describes the `protocol used by OpenSSH's ssh-agent`_.
+
+Top_
+
+
+Why each region must use their own public keys
+==============================================
+
+The region staff team are responsible of the virtual machines instantiated on their
+servers. Therefore each organisation should have the control of who access the virtual
+machines for support purposes and set and enforce the corresponding policy. It is
+not possible if the public keys are shared among all the organisations. Additionally,
+it is also extremely insecure and a problem when an organisation left the federation.
+
```

Comment:
Perhaps it would be more clear to speak about regions instead of organizations (when a region left the federantion, each region should have,,, not shared between regions...)